documentation/src/main/markdown/runningdetect/component-location-analysis.md

```diff
@@ -4,105 +4,65 @@ Enable this feature by adding --detect.component.location.analysis.enabled=TRUE
 
 When enabled, [company_name] [solution_name] creates an output file in the scan subdirectory of the output directory with the name 'components-with-locations.json' which identifies the declaration locations (filepath, line number and column position) of the version of open source components found in the scanned project.
 
-* As Detect shuts down, by default, it performs cleanup operations which include deleting the component location analysis file. You can disable clean up by setting --detect.cleanup=false.
+<note type="note">By default, when [company_name] [solution_name] shuts down, it performs cleanup operations which include deleting the component location analysis file. You can disable clean up by setting `--detect.cleanup=false`.</note>
 
 ## Requirements and Limitations
 
 * Component Location Analysis is available under [company_name] [solution_name] “Limited Customer Availability (LCA)”.
 * A limited subset of Detector Types support this feature.
-    * The currently supported Detectors as of 9.2.0 are: NPM, MAVEN, GRADLE, NUGET, GO_MOD, GO_DEP, GO_GRADLE, GO_VENDOR, GO_VNDR, PIP, POETRY, and CONDA.
-* The currently supported scan modes as of 8.11.0 are: Rapid/Stateless and offline.
-    * Rapid/Stateless Scan mode requires [blackduck_product_name] policies.
-        * Only components that violate policies will be included in the analysis. If no policies are violated or there are no defined policies, then component location analysis is skipped.
+    * Detectors supported as of 9.2.0 are: NPM, MAVEN, GRADLE, NUGET, GO_MOD, GO_DEP, GO_GRADLE, GO_VENDOR, GO_VNDR, PIP, POETRY, CONDA, and YARN.
```

Comment:
Should it say "as of 9.6.0" for line above and below? I don't know if the version needs to be stated.

Reply:
I am inclined to agree and remove the version. We capture that in the release notes and the properties for most things.

Reply:
Yes, we should remove the version.

Reply:
Done.